docs/usage/commands.md

```diff
@@ -22,6 +22,11 @@ It can work over a local repository or a list of GitHub organizations.
 
 **source{d} CE** will download and install Docker images on demand. Therefore, the first time you run some of these commands, they might take a bit of time to start up. Subsequent runs will be faster.
 
+Once **source{d} CE** has been initialized, it will automatically open the web UI.
+If the UI is not opened automatically, you can use [`sourced web`](#sourced-web) command, or visit http://localhost:8088.
```

Comment:
Should it be `127.0.0.1`? https://github.com/src-d/sourced-ce/issues/116

Reply:
Btw - it would be nice to have `--no-browser` option. I forgot to mention in friction log that this command launched a browser without tab `localhost:8088` - so I had to copy-paste it anyway.

Reply:
Thanks for noticing that it should use `127.0.0.1` instead of `localhost`
About the second comment, it should be handled on a separate issue, but:
- what would be the use case of `--no-browser`? when would you want to avoid opening the UI?
- is the browser always opened without url? which browsers do you have? which OS? what happens if you run `xdg-open http://127.0.0.1:8081` in your Linux (or `open http://127.0.0.1:8081` in Darwin)?

Reply:
> what would be the use case of --no-browser? when would you want to avoid opening the UI?

Like `jupyter` - some kind of server application - you launch it, expose port, so external people can connect.

> is the browser always opened without url? 
* Yes, somehow something is misconfigured on my PC
> which browsers do you have? 
* Chromium `Version 74.0.3729.169 (Official Build) (64-bit)`
> which OS? 
```LSB Version:	core-9.20160110ubuntu0.2-amd64:core-9.20160110ubuntu0.2-noarch:printing-9.20160110ubuntu0.2-amd64:printing-9.20160110ubuntu0.2-noarch:security-9.20160110ubuntu0.2-amd64:security-9.20160110ubuntu0.2-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
```
> what happens if you run xdg-open http://127.0.0.1:8088 in your Linux (or open http://127.0.0.1:8088 in Darwin)?
* browser is opened with empty tab

Reply:
So that is the problem. `sourced web` relies on `xdg-open`, which is not working in your PC, maybe because of a misconfigured Chrome as explained by https://askubuntu.com/questions/499200/xdg-open-opens-chrome-but-does-not-load-the-url
Could you confirm?

Reply:
Yes, it was misconfigured. Thank you for fix!